build/lib/extractor.ts

```diff
@@ -467,13 +467,18 @@ class PackExtractor {
                                 img: ImageFilePath;
                                 prototypeToken: DeepPartial<foundry.data.PrototypeTokenSource>;
                             } = docSource;
+                            const ringSetting = docSource.prototypeToken.ring;
                             withToken.prototypeToken = { name: docSource.prototypeToken.name };
                             // Iconics have special tokens
                             if (withToken.img?.includes("iconics")) {
                                 withToken.prototypeToken.texture = {
                                     src: withToken.img.replace("Full", "") as ImageFilePath,
                                 };
                             }
+                            // If using dynamic ring, preserve enabled status and subject texture
+                            if (ringSetting?.enabled) {
```

Comment:
```suggestion
                            const ringSetting = docSource.prototypeToken.ring;
                            if (ringSetting?.enabled) {
```

Reply:
I've tried this, but line 471 clobbers the prototypeToken variable. withToken is actually not a destructuring, its a type narrow.